persistence/proto/src/main/scala/net/liftweb/proto/ProtoUser.scala

```diff
@@ -864,8 +864,10 @@ trait ProtoUser {
       }
     }
 
+    val emailElemId = nextFuncName
+    S.appendJs(Focus(emailElemId))
     val bind =
-      ".email" #> FocusOnLoad(<input type="text" name="username"/>) &
+      ".email" #> <input type="text" name="username" id={emailElemId}/> &
```

Comment:
I have tried some non working stuff combining onload with SHtml.onEvent and also with SHtml.ajaxInvoke calls but haven't got anything working that makes the email input get focus on page load. 
There is a very simple solution that works in html5 browsers (i.e autofocus) but I guess the point is to support older browsers also.
Could you elaborate a bit more on what you had in mind  ?